contracts/proxy/src/contract.rs

```diff
@@ -417,14 +451,26 @@ pub fn execute_request_update_guardians(
 
             PENDING_GUARDIAN_ROTATION.save(
                 deps.storage,
-                &GuardiansUpdateRequest::new(r.guardians, r.new_multisig_code_id, &env.block),
+                &GuardiansUpdateRequest::new(
+                    r.guardians.clone(),
+                    r.new_multisig_code_id,
+                    &env.block,
+                ),
             )?;
 
-            Ok(Response::new().add_attribute("action", "Request to update guardians created"))
+            let event = Event::new("vectis.proxy.v1.MsgRequestUpdateGuardians")
+                .add_attribute("contract_address", env.contract.address)
```

Comment:
why do we have this contract address in these events in the proxy particularly? Can parse in event metadata, not attributes?